src/styles/blocks/buttonBuy.scss

```diff
@@ -0,0 +1,21 @@
+.buttonBuy {
+  color: #fff;
+  background-color: $main-button-color;
```

Comment:
The variable '$main-button-color' is used here. Ensure that this variable is defined in your SCSS files. If it's not defined, you need to define it or correct the variable name.